website/source/docs/connect/proxies.html.md

```diff
@@ -113,33 +113,33 @@ until at least 1.5.0 but it's highly recommended to switch to using
 
 #### Proxy Parameters
 
- - `destination_service_name` `string: <required>` - Specifies the _name_ of the
+ - `destination_service_name` `(string: <required>)` - Specifies the _name_ of the
    service this instance is proxying. Both side-car and centralized
    load-balancing proxies must specify this. It is used during service
    discovery to find the correct proxy instances to route to for a given service
    name.
 
- - `destination_service_id` `string: <optional>` - Specifies the _ID_ of a single
+ - `destination_service_id` `(string: "")` - Specifies the _ID_ of a single
    specific service instance that this proxy is representing. This is only valid
    for side-car style proxies that run on the same node. It is assumed that the
    service instance is registered via the same Consul agent so the ID is unique
    and has no node qualifier. This is useful to show in tooling which proxy
    instance is a side-car for which application instance and will enable
    fine-grained analysis of the metrics coming from the proxy.
 
- - `local_service_address` `string: <optional>` - Specifies the address a side-car
+ - `local_service_address` `(string: "")` - Specifies the address a side-car
    proxy should attempt to connect to the local application instance on.
    Defaults to 127.0.0.1.
 
- - `local_service_port` `int: <optional>` - Specifies the port a side-car
+ - `local_service_port` `(int: <required>)` - Specifies the port a side-car
```

Comment:
I think this is the only one that switched from optional to required; just want to make sure it isn't a mistake.

Reply:
Yeah it is optional. This is different form the upstream bind port above which is required currently.

If not provided we default to the service's advertised port as noted in the comment.

Reply:
Ah glad you queried that @judithpatudith ! Can't think why I thought it needed changing, does this effect line 81 above also? Will look back at what I was doing also.

I _think_ I might have thought 'this should be optional' as I don't think normal proxies require it, but sidecars do? Is that the difference here?

Reply:
Cool, I've reverted that `required` back to `optional` here